lib/promptConfig.js

```diff
@@ -1,87 +1,55 @@
 import chalk from 'chalk';
-import execa from 'execa';
-import inquirer from 'inquirer';
 import getConfig from './getConfig';
 
-const promptConfig = {
-  config: () => getConfig(),
+const config = () => getConfig();
```

Comment:
I just saw it. I guess we can remove that and call it directly from `getConfig()`